tables/rx_potency/uraki23-pot.csv

```diff
@@ -0,0 +1,229 @@
+Uraki23,Imdevimab+Casirivimab,hCoV-19/Japan/TY41-795-P0/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Imdevimab+Casirivimab,hCoV-19/USA/MD-HP40900-PIDYSWHNUB-P2/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Tixagevimab,Wuhan-Hu-1,Supplemental Figure 2A,Virus isolate,IC50,24.0,1,50000,1,ng/ml,2023-03-17
+Uraki23,Tixagevimab,hCoV-19/Japan/UT-NCD1288-2N/2022,Supplemental Figure 2A,Virus isolate,IC50,21447,1,50000,1,ng/ml,2023-03-17
+Uraki23,Tixagevimab,hCoV-19/Japan/TY41-795-P0/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Tixagevimab,hCoV-19/USA/MD-HP40900-PIDYSWHNUB-P2/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Cilgavimab,Wuhan-Hu-1,Supplemental Figure 2A,Virus isolate,IC50,115.4,1,50000,1,ng/ml,2023-03-17
+Uraki23,Cilgavimab,hCoV-19/Japan/UT-NCD1288-2N/2022,Supplemental Figure 2A,Virus isolate,IC50,117.5,1,50000,1,ng/ml,2023-03-17
+Uraki23,Cilgavimab,hCoV-19/Japan/TY41-795-P0/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Cilgavimab,hCoV-19/USA/MD-HP40900-PIDYSWHNUB-P2/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Tixagevimab+Cilgavimab,Wuhan-Hu-1,Supplemental Figure 2A,Virus isolate,IC50,57.7,1,50000,1,ng/ml,2023-03-17
+Uraki23,Tixagevimab+Cilgavimab,hCoV-19/Japan/UT-NCD1288-2N/2022,Supplemental Figure 2A,Virus isolate,IC50,209.1,1,50000,1,ng/ml,2023-03-17
+Uraki23,Tixagevimab+Cilgavimab,hCoV-19/Japan/TY41-795-P0/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Tixagevimab+Cilgavimab,hCoV-19/USA/MD-HP40900-PIDYSWHNUB-P2/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,S309,Wuhan-Hu-1,Supplemental Figure 2A,Virus isolate,IC50,1584,1,50000,1,ng/ml,2023-03-17
+Uraki23,S309,hCoV-19/Japan/UT-NCD1288-2N/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,S309,hCoV-19/Japan/TY41-795-P0/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,S309,hCoV-19/USA/MD-HP40900-PIDYSWHNUB-P2/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Bebtelovimab,Wuhan-Hu-1,Supplemental Figure 2A,Virus isolate,IC50,26.4,1,50000,1,ng/ml,2023-03-17
+Uraki23,Bebtelovimab,hCoV-19/Japan/UT-NCD1288-2N/2022,Supplemental Figure 2A,Virus isolate,IC50,25.3,1,50000,1,ng/ml,2023-03-17
+Uraki23,Bebtelovimab,hCoV-19/Japan/TY41-795-P0/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Bebtelovimab,hCoV-19/USA/MD-HP40900-PIDYSWHNUB-P2/2022,Supplemental Figure 2A,Virus isolate,IC50,50000,1,50000,1,ng/ml,2023-03-17
+Uraki23,Remdesivir,Wuhan-Hu-1,Supplemental Figure 2B,Virus isolate,IC50,800,1,50000,1,µM,2023-03-17

```

Comment:
For chemical drugs we should use EC50 instead of IC50. @KaimingTao Can you confirm this? I couldn't find the document stated this but the previous studies are EC50 (e.g. https://github.com/hivdb/covid-drdb-payload/blob/master/tables/rx_potency/agostini18-pot.csv).